lib/glimesh_web/live/user_settings/components/channel_settings_live.html.leex

```diff
@@ -167,6 +179,31 @@
         </div>
     </div>
 </div>
+
+<h2 class="mt-4"><%= gettext("Homepage") %></h2>
+<p><%= gettext("Glimesh is now showing your channels on the homepage! It's a completely random process, however you must opt-in and you must have completed the following qualifications first.") %></p>
+<div class="row mb-4">
+    <div class="col-sm-6">
+        <strong><%= gettext("Homepage Qualifications") %></strong>
+        <ul>
+            <li><%= gettext("Show on Homepage must be checked") %></li>
+            <li><%= gettext("Must have streamed for at least 10 hours") %></li>
```

Comment:
Possibly to clarify any confusion, unless I am the one who is confusion.
```suggestion
            <li><%= gettext("Must have streamed for at least 10 hours total on Glimesh") %></li>
```

Reply:
I would say the message on line 192 would clarify this one but some people might misunderstand. Especially in translations. :)